README.md

```diff
@@ -667,9 +667,7 @@ be augmented with Tari specific settings. Using the wizard, create the following
     - `Secure connection (TLS)`: Check/Uncheck (based on the pool requirements).
     - `keepalive`: Check.
     - `nicehash`: Uncheck.
-    - `User`: This must be your own mainnet wallet address, or you can use this address to donate to Monero:
-       - Public mainnet address at https://www.getmonero.org/get-started/contributing/
-         `888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H`
+    - `User`: This must be your own mainnet wallet address, or you can use the defaults in the configuration file:
```

Comment:
This _"Pool mining with Self-Select"_ example configuration is working properly and should not be changed

Reply:
Whoops, will revert that line.